frontend/tests/features/collection.test.ts

```diff
@@ -56,9 +61,11 @@ describe("Collection", () => {
   });
 });
 
-async function createCollection(): Promise<string> {
+async function createCollection(unique?: string): Promise<string> {
```

Comment:
discussion: should we update the API to accept an optional collection object?

E.g.,

```ts
interface Collection {
  name: string
  description: string
  ...
}

async function createCollection(collection?: Collection = TEST_COLLECTION) {
  ...
}

// callsite
const name = TEST_COLLECTION + timestamp

createCollection({ ...TEST_COLLECTION, name })

...

await expect(page).not.toHaveSelector(
  getText(name)
)
```

Reply:
How about a Partial<Collection> that overrides the TEST_COLLECTION?

Reply:
That sounds even better! 🔥